frontend/src/metabase/collections/components/CollectionHeader/CollectionHeader.styled.tsx

```diff
@@ -21,6 +21,7 @@ export const HeaderActions = styled.div`
   display: flex;
   margin-top: 0.5rem;
   align-self: start;
+  gap: 0.5rem;
```

Comment:
Use gap the same as https://github.com/metabase/metabase/pull/23833/commits/bf2e2d6625657aee776ccdaab61c0d9c04f1b289#diff-13a900a69321a6b06dbe132e97c09e50cc55b358de6031e32844ecb04eeccbc0R158